### DISCUSSION TITLE
daly bus 16s not working with usb to rs485 adapter

### DISCUSSION THREAD

**Discussion by mpuff** [Louisvdw/dbus-serialbattery#579] [General]
i changed from the uart adapter to the rs485 to usb adapter from daly, now i don't see the battery in Venus os.
do I need something change in the config?
on the uart, I have now plugged in the bluetooth adapter permanently, with that, I can connect to my phone, this one is working, I did a test.
I use a raspberry pi with the Venus os image

**Comment by mr-manuel** [comment]
Did you troubleshoot? You can find in the wiki how to do that.

**Reply by mpuff** [reply]
I tried with that command:
tail -f /data/log/dbus-serialbattery.*/current | tai64nlocal

but I get no respond from bms

**Reply by mr-manuel** [reply]
Please post pictures, where we can see all the cabling from the USB adapter to the serial port

**Comment by teefixx** [comment]
I use three RS485/USB adapters: one with FTDI chip and two from daly with CH341 chip:

it probably depends on the RS485 / USB adapter you have, but for the original sent from daly adapter, I had to add the following line into /etc/udev/rules.d/serial-starter.rules
        - [ ] #Daly CH341	Chip
        - [ ] ACTION=="add", ENV{ID_BUS}=="usb", ENV{ID_MODEL}=="USB Serial",                 ENV{VE_SERVICE}="rs485"

for the one with the FTDI chip:
        - [ ] #RS485 this is for an adapter with FTDI chip
        - [ ] ACTION=="add", ENV{ID_BUS}=="usb", ENV{ID_MODEL}=="USB-RS485-WE",               ENV{VE_SERVICE}="rs485"

I also had to rename my adapter with the FTDI chip (not the daly on) to „USB-RS485-WE“ with the original FTDI software...

hope this helps
